solidity-v1/dashboard/src/css/allocation-progress-bar.less

```diff
@@ -0,0 +1,31 @@
+.allocation-progress-bar {
+  token-allocation-data {
+  }
+
+  &__token-allocation {
+  }
+
+  &__progress-bar-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__progress-bar-wrapper {
+    margin: 0 !important;
+    max-width: 85%;
+  }
+
+  &__allocation-percentage-value {
+  }
```

Comment:
Unnecessary class.

Reply:
Fixed in b8a45ea2b7abd6ea27ffc29dad060a3702bd877d